.github/scripts/verify_arbiter_approvals.py

```
@@ -0,0 +1,147 @@
+"""Verify that arbiter approvals are present for a PR."""
+
+import argparse
+import json
+import subprocess
+import sys
+
+ARBITER_LABEL_PREFIX = "arbiter/"
+
+
+def _run(cmd, check=True):
+    result = subprocess.run(cmd, capture_output=True, text=True)  # lgtm[py/command-line-injection] - cmd is always a fixed-shape list (never shell=True); pr-number is a validated positive int
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)